tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/AlfrescoStackInitializer.java

```diff
@@ -0,0 +1,139 @@
+package org.alfresco.elasticsearch;
+
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.support.TestPropertySourceUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.lifecycle.Startables;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+public class AlfrescoStackInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext>
+{
+
+    public static Network network;
+
+    public static GenericContainer alfresco;
+
+    public static ElasticsearchContainer elasticsearch;
+
+    public static GenericContainer liveIndexer;
+
+    private static String ES_CONNECTOR_TAG = "3.0.0-RC1";
+
+    @Override
+    public void initialize(ConfigurableApplicationContext configurableApplicationContext)
+    {
+
+        network = Network.newNetwork();
+
+        alfresco = new GenericContainer("quay.io/alfresco/alfresco-content-repository:latest")
```

Comment:
This version seems odd as well. I think this will be generated by a different project, so at some point in the future (when the repo starts needing different configuration) the tests in this commit will start failing.  I don't have an answer for this, but it seems like we should be getting all our versions from Maven, and I assume we will update those versions for different release branches.

Reply:
solved: now .env is loadrd to read the tags AND `alfresco/alfresco-content-repository:latest` is used, as per other tests. (alfresco/alfresco-content-repository:latest is created by acs scripts on the fly)